scripts/scraper-ng/rules/html-require-macros.js

```diff
@@ -10,9 +12,12 @@ function requireMacros(tree, file, required = []) {
     node =>
       node.type === "text" &&
       node.data &&
-      required.includes(node.data.macroName),
+      required.map(normalizeMacroName).includes(node.data.macroName),
```

Comment:
I have no idea what I'm talking about but... Why isn't this
```suggestion
      required.map(normalizeMacroName).includes(normalizeMacroName(node.data.macroName)),
```

Reply:
`node.data.macroName` is already normalized (see [this bit](https://github.com/mdn/stumptown-content/pull/271/files#diff-e403f425e4ea2799c526b9b5ad22f009R17)).

As for `required.map(normalizeMacroName)`, I did that to preserve the original capitalization in the configuration (see `preset.js`). This way the linting messages can be a bit prettier (e.g., imagine `SomeComplexMacroName required but not found` versus `somecomplexmacroname required but not found`).

Reply:
Got it. I guess you'd have to read the whole code to understand that `node.data` had already gone through the normalization ringer. :)

Also, mayor bonus for being able to produce an error message based on the input rather then the normalized comparison. 
Like those http libs where you can query for an header case-insensitively and it throws and error if it's not present and the error mentions it like the coder mentioned it.

Reply:
Yeah, I very nearly went with the uglier/surprising output, but thought I should write the code I want to see in the world 😆